lib/fog/aws/requests/rds/add_tags_to_resource.rb

```diff
@@ -12,28 +12,43 @@ class Real
         #   * body<~Hash>:
         def add_tags_to_resource(rds_id, tags)
           keys    = tags.keys.sort
-          values  = keys.map {|key| tags[key]}
+          values  = keys.map { |key| tags[key] }
+          resource_name = "arn:aws:rds:#{@region}:#{owner_id}:db:#{rds_id}"
+          %w[us-gov-west-1 us-gov-east-1].include?(@region) ? resource_name.insert(7, '-us-gov') : resource_name
           request({
-              'Action'        => 'AddTagsToResource',
-              'ResourceName'  => "arn:aws:rds:#{@region}:#{owner_id}:db:#{rds_id}",
-              :parser         => Fog::Parsers::AWS::RDS::Base.new,
-            }.merge(Fog::AWS.indexed_param('Tags.member.%d.Key', keys)).
-              merge(Fog::AWS.indexed_param('Tags.member.%d.Value', values)))
+            'Action' => 'AddTagsToResource',
+            'ResourceName' => resource_name,
+            :parser => Fog::Parsers::AWS::RDS::Base.new
+          }.merge(Fog::AWS.indexed_param('Tags.member.%d.Key', keys))
+              .merge(Fog::AWS.indexed_param('Tags.member.%d.Value', values)))
+        end
+
+        def add_tags_to_dbsnapshot(snapshot_name, tags)
```

Comment:
For consistency I think we might want this to be defined in it's own request file? Presumably `lib/fog/aws/requests/rds/add_tags_to_dbsnapshot.rb`. That will make it easier to find in the future, should we need it. It would also be good to see some tests for this and probably also so Mocks to round it out and provide the same level of integration and functionality as the other functions. Does that make sense? What do you think?

Reply:
yes right @geemus 
removed rds snapshot changes from this PR
Will in another PR.